packages/core/admin/admin/src/components/UnstableGuidedTour/Tours.tsx

```diff
@@ -7,18 +15,91 @@ import type { State, Action } from './Context';
 const tours = {
   contentManager: createTour('contentManager', [
     {
-      name: 'TEST',
-      content: () => (
-        <>
-          <div>This is TEST</div>
-        </>
+      name: 'Introduction',
+      content: (Step) => (
+        <Step.Root sideOffset={-36}>
+          <Step.Title
+            id="tours.contentManager.Introduction.title"
+            defaultMessage="Content manager"
+          />
+          <Step.Content
+            id="tours.contentManager.Introduction.content"
+            defaultMessage="Create and manage content from your collection types and single types."
+          />
+          <Step.Actions showSkip />
+        </Step.Root>
       ),
     },
   ]),
 } as const;
 
 type Tours = typeof tours;
 
+/* -------------------------------------------------------------------------------------------------
+ * GuidedTourTooltip
+ * -----------------------------------------------------------------------------------------------*/
+
+type Content = (
+  Step: Step,
+  {
+    state,
+    dispatch,
+  }: {
+    state: State;
+    dispatch: React.Dispatch<Action>;
+  }
+) => React.ReactNode;
+
+export const GuidedTourOverlay = styled(Box)`
+  position: fixed;
+  inset: 0;
+  background-color: rgba(50, 50, 77, 0.2);
+  z-index: 10;
```

Comment:
I am curios, do you have already checked if this z-index is in conflict with others element with the same value?

Reply:
I've only been testing on the content manage list page so far and there is no issue. The overlay zIndex should be higher than every thing except popover.